pkg/solana/chain.go

```diff
@@ -222,7 +226,7 @@ func (v *verifiedCachedClient) GetAccountInfoWithOpts(ctx context.Context, addr
 	return v.ReaderWriter.GetAccountInfoWithOpts(ctx, addr, opts)
 }
 
-func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.Logger) (*chain, error) {
+func newChain(id string, cfg *config.TOMLConfig, ks core.Keystore, lggr logger.Logger, ds sqlutil.DataSource) (*chain, error) {
```

Comment:
When will this be used?

Reply:
It will be used by LogPoller to initialize its ORM. We have the db schemas & some of the orm written now, but haven't added the constructor for LogPoller yet. I could add it as a part of this PR, but it seems more related to the JIRA we have for the main LogPoller runloop than this one. Originally it looked like tx mgr would also need it, but they've since confirmed they won't need to persist anything to db.